linkerd/app/outbound/src/http/logical/policy/route.rs

```diff
@@ -174,15 +186,15 @@ impl<T> filters::Apply for Http<T> {
 
 impl<T> metrics::MkStreamLabel for Http<T> {
     type StatusLabels = metrics::labels::HttpRouteRsp;
-    type DurationLabels = metrics::labels::Route;
+    type DurationLabels = metrics::labels::HttpRoute; // TODO(kate): do we want hostname here?
```

Comment:
right now we'd be adding the hostname label to our durations. if cardinality concerns are going to matter anywhere, they'll be happening here.

on the other hand though, i can imagine this being tremendously useful latency info.

i'd love to hear what people think!

Reply:
i've refrained from adding that label here.

it's still an interesting follow-on / enhancement to consider, but i'll go ahead and mark this as resolved for now.